welcome-message/index.js

```diff
@@ -108,6 +113,13 @@ function hideIfSegmentId () {
 	}
 }
 
+function setTourButton () {
+	const tourButton = fixedEl.querySelector('[data-component="cta-take-tour"]');
+	tourButton.addEventListener('click', () => {
```

Comment:
it won't get called unless `fixedEl.getAttribute('data-component') === 'welcome-banner'` is true